src/Discord.Net.WebSocket/Entities/Interaction/SocketInteraction.cs

```diff
@@ -109,13 +110,14 @@ internal virtual void Update(Model model)
         /// <exception cref="ArgumentOutOfRangeException">Message content is too long, length must be less or equal to <see cref="DiscordConfig.MaxMessageSize"/>.</exception>
         /// <exception cref="InvalidOperationException">The parameters provided were invalid or the token was invalid.</exception>
         public abstract Task RespondAsync(string text = null, Embed[] embeds = null, bool isTTS = false,
-            bool ephemeral = false, AllowedMentions allowedMentions = null, RequestOptions options = null, MessageComponent component = null);
+            bool ephemeral = false, AllowedMentions allowedMentions = null, RequestOptions options = null, MessageComponent component = null, Embed embed = null);
 
         /// <summary>
         ///     Sends a followup message for this interaction.
         /// </summary>
         /// <param name="text">The text of the message to be sent</param>
         /// <param name="embeds">A array of embeds to send with this response. Max 10</param>
+        /// <param name="embed">DEPRECTAED single embed to send in followup</param>
```

Comment:
move the `embed ` param tag to the end of the param list and remove `DEPRICATED ` as its not anymore